app/models/attraction.rb

```diff
@@ -22,6 +22,10 @@ class Attraction < ApplicationRecord
                                 :accessibility_information, :operating_company,
                                 :data_provider, :web_urls, :certificates,
                                 :regions
+
+  def category_name
```

Comment:
wenn dies hier der neue AttributeReader wird, dann muss die bestehende Integration von :category_name als `:attr_accessor` geändert werden auf ein `:attr_writer`. Damit würde dann die Methode `def category_name`
 der `:attr_reader` dazu werden

Reply:
mit https://github.com/ikuseiGmbH/smart-village-app-mainserver/pull/68/files wird das hier sogar hinfällig, oder? es gibt ja dann den namen direkt. das kann ich hier vor dem mergen wieder entfernen, oder @acousma82?

Reply:
Nein wird nicht hinfällig. Bitte alles so lassen. Der Kommentar gehört eigentlich in "meinen PR".

Reply:
also lasse ich `attr_accessor :category_name` und habe dann unten

```
def find_or_create_category
    self.category_id = Category.where(name: category_name).first_or_create.id
  end

  def category_name
    category.name
  end
```

??

Reply:
attr_writer :category_name ist richtig
dadurch wird deine methode unten zum reader und meine oben zum writer.

Reply:
Ansonsten überschreibt deine methode meine accesor methode oben.